apps/landing/app/layout.tsx

```diff
@@ -35,7 +35,7 @@ export default function RootLayout({
   children: React.ReactNode;
 }>) {
   return (
-    <html lang="en" className={`${GeistSans.variable} ${GeistMono.variable}`}>
+    <html lang="en" className={`scroll-smooth ${GeistSans.variable} ${GeistMono.variable}`}>
```

Comment:
Do we want to add scroll smooth to the entire marketing site?

Reply:
discussing